.gitattributes

```diff
@@ -1 +1,2 @@
 Coverage/taxa-db.sqlite filter=lfs diff=lfs merge=lfs -text
+*.R filter=lfs diff=lfs merge=lfs -text
```

Comment:
This line is marking all .R files as needing to go through git lfs. Is there a reason this change is here?